data/brands/craft/electronics_repair.json

```diff
@@ -123,6 +123,17 @@
         "name": "uBreakiFix"
       }
     },
+    {
+      "displayName": "Asurion Tech Repair & Solutions",
+      "locationSet": {"include": ["us"]},
+      "tags": {
+        "brand": "Asurion Tech Repair & Solutions",
+        "brand:wikidata": "Q4811938",
+        "craft": "electronics_repair",
+        "electronics_repair": "phone",
```

Comment:
Could the `electronics_repair` tag be expanded if they offer more than just phone repairs? Do they also offer computer / appliance repairs in store?

Reply:
Possibly `electronics_repair=computer;phone`, but I'm not completely sure, not an American.

This is a copy/paste of the uBreakiFix. So if so that can probably be updated too.

Reply:
So asurion lists [appliances](https://www.asurion.com/repairs/appliance/) on there website too, I can't see the same on uBreakiFix.